documentation/docs/api_readme.md

```diff
@@ -1,4 +1,4 @@
 `@cognite/reveal` has three modules:
 - [@cognite/reveal](#module-cognitereveal) is the main module and has the main entry point of Reveal, [`Cognite3DViewer`](#class-cognite3dviewer). CAD models are represented by [`CogniteCadModel`](#class-cognitecadmodel) and points clouds by [`CognitePointCloudModel`](#class-cognitepointcloudmodel).
 - [@congite/reveal/tools](#module-cogniterevealtools) contains a set of tools, e.g. [`AxisViewTool`](#class-axisviewtool), [`GeomapTool`](#class-geomaptool), [`TimelineTool`](#class-timelinetool), [`ExplodedViewTool`](#class-explodedviewtool) and [`HtmlOverlayTool`](#class-htmloverlaytool) that works with `Cognite3DViewer`.
-- [@cognite/reveal/extensions/datasource](#module-cogniterevealextensionsdatasource) allows providing custom data sources for geometry and model metadata.
```

Comment:
I guess we'll have to update this file now :sweat_smile:

Reply:
Yeah, I messed up here, I see. Need to document these changes.